backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/importer/nuernberg/steps/DownloadCsv.kt

```diff
@@ -24,44 +23,53 @@ class DownloadCsv(config: ImportConfig, private val logger: Logger) :
                 getCSVInputStream(),
                 Charsets.UTF_8
             )
-            val records: Iterable<CSVRecord> = CSVFormat.RFC4180.parse(inputCSV)
+            val parser = CSVFormat.RFC4180.builder().setHeader().setSkipHeaderRecord(true).build().parse(inputCSV)
 
-            return getCSVAcceptingStores(records)
+            return getCSVAcceptingStores(parser)
         } catch (e: Exception) {
             logger.error("Unknown exception while downloading data from csv", e)
             throw e
         }
     }
 
-    private fun getCSVAcceptingStores(records: Iterable<CSVRecord>): List<CSVAcceptingStore> {
+    private fun getCSVAcceptingStores(parser: CSVParser): List<CSVAcceptingStore> {
         val stores: MutableList<CSVAcceptingStore> = arrayListOf()
-        records.forEachIndexed { index, record ->
-            val headline = index == 0
-            // at least a name is required
-            if (record[1].isNotEmpty() && !headline) {
-                stores.add(
-                    CSVAcceptingStore(
-                        record[0],
-                        record[1],
-                        record[2],
-                        record[3],
-                        record[4],
-                        record[5],
-                        record[6],
-                        record[7],
-                        record[8],
-                        record[9],
-                        record[10],
-                        record[11],
-                        record[12]
-                    )
-                )
+        parser.records.forEach { record ->
+            val name = record.get("Name")
+            if (name.isBlank()) {
+                throw IllegalArgumentException("CSV record " + record.recordNumber + " has an empty name.")
             }
+
+            val (discountDE, discountEN) =
+                if (parser.headerMap.containsKey("Rabatt")) {
+                    // Legacy format used until April 2024. Can be removed in future releases.
+                    Pair(record.get("Rabatt"), null)
```

Comment:
if i import it with the old csv format i get the `null` as a value in the database. It will be converted to a string